planning/behavior_path_planner/src/scene_module/lane_change/external_request_lane_change_module.cpp

```diff
@@ -371,50 +371,49 @@ std::pair<bool, bool> ExternalRequestLaneChangeModule::getSafePath(
 
   const auto current_lanes = util::getCurrentLanes(planner_data_);
 
-  if (!lane_change_lanes.empty()) {
-    // find candidate paths
-    const auto lane_change_paths = lane_change_utils::getLaneChangePaths(
-      *route_handler, current_lanes, lane_change_lanes, current_pose, current_twist,
-      common_parameters, *parameters_);
-
-    // get lanes used for detection
-    lanelet::ConstLanelets check_lanes;
-    if (!lane_change_paths.empty()) {
-      const auto & longest_path = lane_change_paths.front();
-      // we want to see check_distance [m] behind vehicle so add lane changing length
-      const double check_distance_with_path = check_distance + longest_path.length.sum();
-      check_lanes = route_handler->getCheckTargetLanesFromPath(
-        longest_path.path, lane_change_lanes, check_distance_with_path);
-    }
+  if (lane_change_lanes.empty()) {
+    return std::make_pair(false, false);
+  }
```

Comment:
Same change should be applied to lane_change_module.cpp

Reply:
fixed in [ee94159](https://github.com/autowarefoundation/autoware.universe/pull/2776/commits/ee941590f8e7b90183419921d03822f8f2af778f)